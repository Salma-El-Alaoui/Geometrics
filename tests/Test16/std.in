C c2 2 2 2
R r2 2 2 2 2
L l2 2 2 2 2
SAVE test16-1.txt
LOAD test15-1.txt
SAVE test16-2.txt
EXIT

