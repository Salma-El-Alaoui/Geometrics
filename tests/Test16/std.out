OK
#Element ajoute: c2
OK
#Element ajoute: r2
OK
#Element ajoute: l2
OK
#Fichier test16-1.txt cree.
OK
#Fichier charge
OK
#Fichier test16-2.txt cree.

